src/invoices/InvoiceDetails/utils/utils.js

```diff
@@ -179,6 +139,7 @@ export const handleInvoiceLineErrors = async ({
   requestData = [],
   responses = [],
   showCallout,
+  ky = noop,
```

Comment:
`noop.get()` -> `TypeError: noop.get is not a function`. `ky` it's an object, not the function

Reply:
updated